aten/src/ATen/native/cuda/TensorModeKernel.cu

```diff
@@ -139,8 +139,8 @@ void handle_fused_mode(
     int64_t slice_size,
     int64_t slices) {
   constexpr int num_threads = size / 2;
-  static_assert(num_threads % C10_WARP_SIZE == 0 &&
-                num_threads <= cuda_utils::kCUDABlockReduceMaxThreads, "");
+  TORCH_INTERNAL_ASSERT(num_threads % at::cuda::warp_size() == 0);
+  static_assert(num_threads <= cuda_utils::kCUDABlockReduceMaxThreads, "");
```

Comment:
How does `kCUDABlockReduceMaxThreads` work here when it's defined in terms of `C10_WARP_SIZE`?
https://github.com/pytorch/pytorch/blob/f9422e1c6b5dab15eacbbb0e1d4370b07db1e8e8/aten/src/ATen/native/cuda/block_reduce.cuh#L17

Reply:
fixed